packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -35,6 +35,7 @@ const emit = defineEmits<{
     ]"
     :disabled="props.disabled || props.loading"
     :type="props.type"
+    :aria-label="props.label"
```

Comment:
are you sure that the label will not be read out by the screen reader twice?
maybe aria-labelledby would be better?

Reply:
Good point, we can not use `aria-labelledby` because there is no label when loading.
I changed it to only add the aria label when loading